### ISSUE TITLE
ERROR_FAIL_TO_CONTRIBUTE message prompted after submit a contribute

### ISSUE THREAD

**Issue by QAbest** [daicostarter/issues#33]
ERROR_FAIL_TO_CONTRIBUTE message prompted after submit a contribute

**Comment by crawlregister** [comment]
I need more information, how to reproduce it, are you logged in or not, are you network connected or not, what's the amount you are trying to contributed,  which project (url), and maybe a screenshot.

**Comment by QAbest** [comment]
![bug1_0914](https://user-images.githubusercontent.com/43140416/45917352-a3b27e80-bea7-11e8-9d99-9ac31f2052d4.PNG)

**Comment by QAbest** [comment]
Yes, the issue prompted after I logged in, and the project is like above, the contribute amount is 2.00 ETH, which equals with Soft Cap.

**Comment by crawlregister** [comment]
@QAbest looks like your metamask are not connected with our Ethereum test net.

Please verify  if problem still exists.
